settings.gradle

```diff
@@ -9,25 +9,27 @@ include ':libs:login:WordPressLoginFlow'
 include ':WordPressMocks'
 project(':WordPressMocks').projectDir = new File(rootProject.projectDir, properties.getOrDefault('wp.wordpress_mocks_path', 'libs/mocks') + '/WordPressMocks')
 
-include ':react-native-aztec'
-project(':react-native-aztec').projectDir = new File(rootProject.projectDir, 'libs/gutenberg-mobile/react-native-aztec/android')
+include ':@wordpress_react-native-aztec'
```

Comment:
It just reflects the npm package name `@wordpress/react-native-aztec`. When you autolink a react-native dependency, it will keep the `@` and replace the `/` with an `_`.